subliminal/providers/legendastv.py

```diff
@@ -266,12 +275,19 @@ def get_archives(self, title_id, language_code):
                                             'pack' in archive_soup['class'], 'destaque' in archive_soup['class'],
                                             self.server_url + archive_soup.a['href'][1:])
 
+                data_text = archive_soup.find('p', class_='data').text
+
                 # match rating
-                match = rating_re.match(archive_soup.find('p', class_='data').text)
+                match = rating_re.match(data_text)
                 archive.downloads = int(match.group('downloads'))
                 if match.group('rating') is not None:
                     archive.rating = int(match.group('rating'))
 
+                # timestamp
+                match = timestamp_re.search(data_text)
+                if match:
```

Comment:
Location is Brazil - São Paulo: GMT - 3

Note: during summer, the summer time (end of October to end of February): GMT - 2